Shoko.Server/API/v3/Controllers/SeriesController.cs

```diff
@@ -134,6 +134,22 @@ public ActionResult<List<SeriesRelation>> GetShokoRelationsBySeriesID([FromRoute
                 .ToList();
         }
         
+        /// <summary>
+        /// Get a paginated list of <see cref="Series"/> without local files, available to the current <see cref="User"/>.
+        /// </summary>
+        /// <param name="pageSize">The page size.</param>
+        /// <param name="page">The page index.</param>
+        /// <returns></returns>
+        [HttpGet("WithoutFiles")]
+        public ActionResult<ListResult<Series>> GetSeriesWithoutFiles([FromQuery] [Range(0, 100)] int pageSize = 50, [FromQuery] [Range(1, int.MaxValue)] int page = 1)

```

Comment:
FWIW - better choice for `page` is to use `uint` because that itself inherently doesn't support negative values (and gives another bits worth of value for it)